sssd_test_framework/utils/sssctl.py

```diff
@@ -263,3 +263,26 @@ def domain_status(
         }
 
         return self.host.ssh.exec(["sssctl", "domain-status"] + self.cli.args(args), raise_on_error=False)
+
+    def analyze(
```

Comment:
could be better to call this `analyze_request` and always execute request subcommands? We have an analyze `error` module (https://github.com/SSSD/sssd/commit/ed3726c37fe07aab788404bfa2f9003db15f4210) but no need to test it at the moment

Reply:
Good point. Fixed